ux/drumplayer.java

```diff
@@ -4,10 +4,17 @@
 
 	What kind of methods does it need?
 	- play_drums
-		- it could have some kind of global variable that will increase or decrease the volume and/or speed. 
-		  Maybe one for each? volume and speed?
-		- Will it play drums for a certain amount of time? Will it play drums until it gets some other input?
```

Comment:
For each effect there should be a duration:  music, drums, lights; and some indicator of what intensity level.  Fran is writing 4 pieces that will range from nice to chaotic;  the lights will respond in several levels; as will drums.